JASP-Engine/JASP/R/descriptives.R

```diff
@@ -977,14 +978,24 @@ Descriptives <- function(jaspResults, dataset, options) {
 
 
 .plotMarginal <- function(column, variableName,
-                          rugs = FALSE, displayDensity = FALSE) {
+                          rugs = FALSE, displayDensity = FALSE, binWidthType) {
```

Comment:
```suggestion
                          rugs = FALSE, displayDensity = FALSE, binWidthType = c("doane", "fd")) {
  binWidthType <- match.arg(binWidthType)
```
adds a tiny bit of error handling that should make it easier to reuse this function.